files/fr/web/api/htmliframeelement/contentdocument/index.html

```diff
@@ -0,0 +1,22 @@
+---
+title: HTMLIFrameElement.contentDocument
+slug: Web/API/HTMLIFrameElement/contentDocument
+browser-compat: api.HTMLIFrameElement.contentDocument
+translation_of: 'HTMLIFrameElement.contentDocument'
+---
+<p>Si l'iframe et le document parent de l'iframe sont de la <a href="/fr/docs/Web/Security/Same-origin_policy">même origine</a>, <code>HTMLIFrameElement.contentDocument</code> retourne un <a href="/fr/docs/Web/API/Document" title="L'interface « Document » représente toute page web chargée dans le navigateur en tant que point d'entrée dans le contenu de la page, à savoir l'arbre DOM."><code>Document</code></a> (c'est à dire le document actif dans le contexte de navigation imbriqué du cadre. Sinon, il retourne <code>null</code>.</p>
+
+<h2 id="example_of_contentDocument">Exemple d'utilisation de <code>contentDocument</code></h2>
```

Comment:
In order to be consistent w/ other regions of MDN, I'd simply use "Exemples" (or "Exemple" if you prefer (this form is less used).

Reply:
Makes sense, I used "Exemples" as it's more future-proof, in case we add more examples :)
Fixed in https://github.com/mdn/translated-content/pull/1791/commits/28d19f17ce6a22172dcee75cbf7bf58f7335586f